test/spec/provider/camunda/ErrorsPropsSpec.js

```diff
@@ -0,0 +1,578 @@
+    }));
+
+
+    it('should exist on external service task', inject(function(elementRegistry, selection) {
+
+      // given
+      var serviceTask = elementRegistry.get('ServiceTask_1');
+
+      // when
+      selection.select(serviceTask);
+
+      var errorGroup = getErrorsGroup(container);
+
+      // then
+      expect(domClasses(errorGroup).has('bpp-hidden')).to.be.false;
+    }));
+
+  });
+
+
+  describe('error entries', function() {
+
+    it('should display placeholder if there are no errors declared', inject(function(elementRegistry, selection) {
+
+      // given
+      var serviceTask = elementRegistry.get('ServiceTask_1');
+      selection.select(serviceTask);
+
+      // when
+      var noErrorLabel = domQuery('[data-entry="error-placeholder"] span', container);
+
+      // then
+      expect(noErrorLabel).to.exist;
+      expect(noErrorLabel.textContent).to.eql('No errors defined.');
+    }));
+
+
+    describe('collapsible', function() {
+
+      it('should display no error title', inject(function(elementRegistry, selection) {
+
+        // given
+        var serviceTask = elementRegistry.get('ServiceTask_2');
+        selection.select(serviceTask);
+
+        // when
+        var collapsible = getCollapsibleContainer(container, 0);
+        var collapsibleTitle = getCollapsibleTitle(collapsible);
+
+        // then
+        expect(collapsibleTitle.textContent).to.eql('No Error referenced');
+      }));
+
+
+      it('should display error title', inject(function(elementRegistry, selection) {
+
+        // given
+        var serviceTask = elementRegistry.get('ServiceTask_2');
+        selection.select(serviceTask);
+
+        // when
+        var collapsible = getCollapsibleContainer(container, 1);
+        var collapsibleTitle = getCollapsibleTitle(collapsible);
+
+        // then
+        expect(collapsibleTitle.textContent).to.eql('Error_1');
+      }));
+
+
+      it('should display error code', inject(function(elementRegistry, selection) {
+
+        // given
+        var serviceTask = elementRegistry.get('ServiceTask_2');
+        selection.select(serviceTask);
+
+        // when
+        var collapsible = getCollapsibleContainer(container, 2);
+        var collapsibleTitle = getCollapsibleTitle(collapsible);
+
+        // then
+        expect(collapsibleTitle.textContent).to.contain('(code = 123)');
+      }));
+
+
+      it('should display throw expression', inject(function(elementRegistry, selection) {
+
+        // given
+        var serviceTask = elementRegistry.get('ServiceTask_2');
+        selection.select(serviceTask);
+
+        // when
+        var collapsible = getCollapsibleContainer(container, 2);
+        var collapsibleDescription = getCollapsibleDescription(collapsible);
+
+        // then
+        expect(collapsibleDescription.textContent).to.eql('${expression}');
+      }));
+
+    });
+
+
+    describe('create camunda:ErrorEventDefinition', function() {
```

Comment:
These (and other as for the errors) tests are more testing the DOM changes (is collapsible existent, has select box this amount of entries). I think we should also check whether the updates were made on the business object level.

```js
describe('create camunda:ErrorEventDefinition', function() {

  describe('in the DOM', function () {
    
    it('should do');


    it('should undo');


    it('should redo');

  });


  describe('on the businessObject', function () {
    
    it('should do');


    it('should undo');


    it('should redo');

  });

});
```

Reply:
Will add businessObject-related tests.

Reply:
cool!